docker-compose.yml

```diff
@@ -13,14 +13,14 @@ services:
       spring.config.location: classpath:/application-custom.yaml
 
       # Enable these for MySQL
-      spring.datasource.url: 'jdbc:mysql://hapi-fhir-mysql:3306/hapi'
-      spring.datasource.driverClassName: com.mysql.jdbc.Driver
-      spring.jpa.properties.hibernate.dialect: org.hibernate.dialect.MySQL5InnoDBDialect
+      # spring.datasource.url: 'jdbc:mysql://hapi-fhir-mysql:3306/hapi'
```

Comment:
Wy are changes to the Dockerfile part of this pull request?

Reply:
The dev branch was not in sync with master branch,which is why i synced the dev and master branches.